staging/kos/cmd/attachment-tput-driver/README.md

```diff
@@ -282,10 +296,12 @@ The driver has two kinds of output: data files and `glog` logging.
 
 For the data files, the driver creates a directory under the current
 working directory.  The new directory's name is the same as the "run
-ID" (which is checked for safety).  The driver writes three data files
-into that directory.  One holds the result of scraping the driver's
-Prometheus metrics.  The other two are the size distributions
-described above, in CSV format.
+ID" (which is checked for safety).  The driver always writes three data
```

Comment:
"always writes three" could be interpreted to mean "always write exactly three", which is not what is meant here.  How about "The driver writes at least three data files..."?

Reply:
Fixed in 49677ba.